packages/server/patches/geckodriver+4.4.2.patch

```diff
@@ -0,0 +1,311 @@
+-
+-Type: `string[]`<br />
+-Default: `[]`
+-
+-### `binary`
+-Path to the Firefox binary.
+-
+-Type: `string`
+-
+-### `connectExisting`
+-Connect to an existing Firefox instance.
+-
+-Type: `boolean`<br />
+-Default: `false`
+-
+-### `host`
+-Host IP to use for WebDriver server.
+-
+-Type: `string`<br />
+-Default: `0.0.0.0`
+-
+-### `jsdebugger`
+-Attach browser toolbox debugger for Firefox.
+-
+-Type: `boolean`<br />
+-Default: `false`
+-
+-### `log`
+-Set Gecko log level [possible values: `fatal`, `error`, `warn`, `info`, `config`, `debug`, `trace`].
+-
+-Type: `string`
+-
+-### `logNoTruncated`
+-Write server log to file instead of stderr, increases log level to `INFO`.
+-
+-Type: `boolean`
+-
+-### `marionetteHost`
+-Host to use to connect to Gecko.
+-
+-Type: `boolean`<br />
+-Default: `127.0.0.1`
+-
+-### `marionettePort`
+-Port to use to connect to Gecko.
+-
+-Type: `number`<br />
+-Default: `0`
+-
+-### `port`
+-Port to listen on.
+-
+-Type: `number`
+-
+-### `profileRoot`
+-Directory in which to create profiles. Defaults to the system temporary directory.
+-
+-Type: `string`
+-
+-### `geckoDriverVersion`
+-A version of Geckodriver to start. See https://github.com/mozilla/geckodriver/releases for all available versions, platforms and architecture.
+-
+-Type: `string`
+-
+-### `customGeckoDriverPath`
+-Don't download Geckodriver, instead use a custom path to it, e.g. a cached binary.
+-
+-Type: `string`<br />
+-Default: `process.env.GECKODRIVER_PATH`
+-
+-### `cacheDir`
+-The path to the root of the cache directory.
+-
+-Type: `string`<br />
+-Default: `process.env.GECKODRIVER_CACHE_DIR || os.tmpdir()`
+-
+-# Other Browser Driver
+-
+-If you also look for other browser driver NPM wrappers, you can find them here:
+-
+-- Chrome: [giggio/node-chromedriver](https://github.com/giggio/node-chromedriver)
+-- Microsoft Edge: [webdriverio-community/node-edgedriver](https://github.com/webdriverio-community/node-edgedriver)
+-- Safari: [webdriverio-community/node-safaridriver](https://github.com/webdriverio-community/node-safaridriver)
+-
+----
+-
+-For more information on WebdriverIO see the [homepage](https://webdriver.io).
+diff --git a/node_modules/geckodriver/dist/index.js b/node_modules/geckodriver/dist/index.js
+index 2367e0b..18be0e1 100644
+--- a/node_modules/geckodriver/dist/index.js
++++ b/node_modules/geckodriver/dist/index.js
+@@ -1,11 +1,11 @@
+ import cp from 'node:child_process';
+-import logger from '@wdio/logger';
++import debugModule from 'debug';
+ import { download as downloadDriver } from './install.js';
+ import { hasAccess, parseParams } from './utils.js';
+ import { DEFAULT_HOSTNAME } from './constants.js';
+-const log = logger('geckodriver');
++const debug = debugModule('cypress-verbose:server:browsers:geckodriver');
+ export async function start(params) {
+-    const { cacheDir, customGeckoDriverPath, ...startArgs } = params;
++    const { cacheDir, customGeckoDriverPath, spawnOpts, ...startArgs } = params;
```

Comment:
Looks like your contribution has also been released so we can probably remove this portion of the patch.

Reply:
im gonna patch this in with https://github.com/cypress-io/cypress/issues/30301 since I need to update it anyway